server.py

```diff
@@ -4,15 +4,25 @@
 socket = socket.socket(
     socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
     )
-
 socket.bind(ADDR)
-
 socket.listen(1)
 
 
 def response_ok():
-    response_message = "HTTP 200 OK"
-    return response_message
+    return conn.sendall(
+        "HTTP/1.1 200 OK\n"
+        "Content-Type: text/html; charset=utf-8\n"
+        "<!DOCTYPE html>\n"
+        "<html>\n"
+        "\t<head>\n"
+        "\t\t<title>Status 200 OK</title>\n"
+        "\t</head>\n"
+        "\t<body>\n"
+        "\t\t<p>HTTP:200 OK response</p>\n"
+        "\t</body>\n"
+        "</html>\n"
+        )
```

Comment:
where does `conn` come from?  It's not an argument you pass to this function.  Are you relying on it being in the scope where this function is called?  That's not going to go well for you.  Besides, this function is not meant to *send* the response to the client.  It's only purpose is to *construct* the response.